clients/vscode/src/chat/ChatViewProvider.ts

```diff
@@ -25,13 +25,16 @@ import type { ServerInfo } from "tabby-agent";
 import type { AgentFeature as Agent } from "../lsp/AgentFeature";
 import { createClient } from "./chatPanel";
 import { GitProvider } from "../git/GitProvider";
-
+import { getLogger } from "../logger";
+import { contributes } from "../../package.json";
+import { parseKeybinding } from "../util/KeybindingParser";
 export class ChatViewProvider implements WebviewViewProvider {
   webview?: WebviewView;
   client?: ServerApi;
   private pendingMessages: ChatMessage[] = [];
   private pendingRelevantContexts: Context[] = [];
   private isChatPageDisplayed = false;
+  private isMac = process.platform === "darwin";
```

Comment:
Do not use Node.js api `process` directly, it will be replaced with a polyfill implementation when the code is built for the browser.

Reply:
how about using  env.appRoot && env.appRoot[0] !== '/' to check isWin or not